src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerConnection.java

```diff
@@ -358,4 +358,19 @@ CallableStatement prepareCall(String sql, int nType, int nConcur, int nHold,
      *        boolean value for 'useFmtOnly'.
      */
     void setUseFmtOnly(boolean useFmtOnly);
+    
+    /**
+     * Returns the current flag value for delayLoadingLobs.
+     *
+     * @return 'delayLoadingLobs' property value.
+     */
+    boolean getDelayLoadingLobs();

```

Comment:
Wouldn't this be a breaking change? Since we're adding new methods to a public interface, any user who has been using ISQLServerConnection to implement their own connection class would be getting a compilation error now.

Reply:
Technically I guess, I'm not sure if anyone actually implements our interfaces.